.github/dependabot.yml

```diff
@@ -0,0 +1,30 @@
+version: 2
+
+updates:
+  - package-ecosystem: github-actions
+    directory: "/"
+    schedule:
+      interval: weekly
+      day: sunday
+      time: "20:00"
+    groups:
+      github-actions:
+        patterns:
+          - "*"
+    reviewers:
+      - alma/squad-devx
+
+  - package-ecosystem: pip
```

Comment:
Who is the owner of this? Because these are only useful if they are reviewed, and a new release is created